MovingBlocks/movingblocks.github.com
# Implementing Pagination

Currently, no UI has been added to navigate through paginated pages. To test out if it's working or not I added some sample blog posts.

To test, - 

1. Run `gatsby develop`
2. Navigate to `/blog` and `/blog/2`

Even though the code is working as expected, I'm getting an error regarding GraphQL query not getting arguments to paginate. 

Edit: The GraphQL error is fixed. Also added a basic link to allow the user to navigate through paginated pages.
